File: Models/Sofmaster.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace AniDAAPI.Models
{
    public partial class Sofmaster
    {
        public Sofmaster()
        {
            Softrans = new HashSet<Softran>();
        }

        public int SofmasterId { get; set; }
        public string Softype { get; set; }

        public virtual ICollection<Softran> Softrans { get; set; }
    }
}

File: DA_CustomerWebPortal/lib/Model/SOFMaster.dart
class SOFMaster {
  int sofmasterID;
  String softype;

  SOFMaster({
    this.sofmasterID,
    this.softype,
  });

  factory SOFMaster.fromJson(Map<String, dynamic> json) {
    return SOFMaster(
      sofmasterID: json['sofmasterId'],
      softype: json['softype'],
    );
  }
}
